docs/application/tizen-studio/release-notes/3-7-release-notes.md

```diff
@@ -0,0 +1,58 @@
+# Tizen Studio 3.7 Release Notes
+
+- Release Date: Mar 31, 2020
+
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - JDK has been bundled with Tizen Studio removing the dependency of Java being installed in the system.
+  - Support for Chrome 80 has been added.
+  - Log Viewer has been re-enabled in the IDE.
```

Comment:
Please change "the IDE" to "Tizen IDE".
For more information, see https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md.